scalableminds/webknossos
# Warn about overwrite mode when no voxels were changed during brush

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- use brush or lasso tool to change voxels
- use overwrite mode "overwrite empty" and test that a toast appears when only brushing over already existing voxels:
![image](https://github.com/scalableminds/webknossos/assets/2486553/4bc9c883-72a2-471e-b892-84e4f1c5803b)
- brush into an empty area -> the toast should disappear

### Issues:
- fixes https://scm.slack.com/archives/C5AKLAV0B/p1704725599769159

------
